test/handlers-option.js

```diff
@@ -19,28 +19,33 @@ test('handlers option', function(t) {
     'should override default handler'
   )
 
-  var mdastWithMetaNode = u('paragraph', [
-    u('meta-node', 'meta'),
+  var customMdast = u('paragraph', [
+    u('custom', 'with value'),
+    u('custom', [u('img', {src: 'with-children.png'})]),
```

Comment:
ahhh, sorry for this mistake, I was thinking in hast, not mdast!

```suggestion
    u('custom', [u('image', {url: 'with-children.png'})]),
```